### ISSUE TITLE
The confidence doesnt update?

### ISSUE THREAD

**Issue by pointlevel** [andrewjfreyer/monitor#308]
Hello Andrew! BIG thanks for your effort in "Monitor". The idea behind it is so excellent.

The thing is that i really cant get it to work as i think its supposed to. I have read the readme-file over and over, but i cant really get a hang of how to best set my parameters. 
My scenario: One monitor as add-on to hassio at the entrance. Another pi zero at the topfloor. Im sending depart and arrive-scans when locking and unlocking my frontdoor. The problem is that the entrance-monitor can stay in 100% the whole day. 
Right now i have "-b -tr" for my entrance-monitor, and "-tad" for my topfloor. The topfloor is responding to the depart/arrival-scans, but the entrance can stay the same for hours. No error-messages, just "idling"i believe..

I really want to use this solution because its idea is the absolutely best so far.

**Comment by andrewjfreyer** [comment]
I'd suggest playing with the -tr option on the main node. Are you using beacons throughout your house? Else, there's no reason to have the -b option. 

Perhaps -tdr for your main node?

**Comment by riggs498** [comment]
I have tried this install. I got the same result. It is like the  pi just forgets what it is doing. I have to restart the service. Then it receives twice.  Then forgets again. It is mainly the master not the slaves.

**Comment by dxdc** [comment]
@riggs498 Just curious, did you try deleting the mqtt topic (from your mqtt broker)? I'm wondering if the retain flag has something to do with it. I saw some similar behavior after changing preferences and re-initializing the topics.

**Comment by riggs498** [comment]
No, I never did get it to work. It kept running in to the problem of the
main board would just stop scanning. I almost had to keep restarting it
every 5 minutes or so. I was getting an error of no hci. Which made no
sense at all. Once I restarted the service. It started working again. Once
an arrival was there. It sent the request. That was it. It was like it need
permission again. When it worked. It worked great. I wish I could get this
to work. I reverted back to the presence.sh. my house has to much rf. This
would of help greatly. Also with this happening. I was getting a lot of
false readings.

**Comment by dxdc** [comment]
Sorry to hear that @riggs498 ... thx for the reply
